backend/proxy/secure_socks_proxy.go

```diff
@@ -21,27 +19,13 @@ import (
 	"golang.org/x/net/proxy"
 )
 
-var (
-	// PluginSecureSocksProxyEnabled is a constant for the GF_SECURE_SOCKS_DATASOURCE_PROXY_SERVER_ENABLED
-	// environment variable used to specify if a secure socks proxy is allowed to be used for datasource connections.
-	PluginSecureSocksProxyEnabled = "GF_SECURE_SOCKS_DATASOURCE_PROXY_SERVER_ENABLED"
-	// PluginSecureSocksProxyClientCert is a constant for the GF_SECURE_SOCKS_DATASOURCE_PROXY_CLIENT_CERT
-	// environment variable used to specify the file location of the client cert for the secure socks proxy.
-	PluginSecureSocksProxyClientCert = "GF_SECURE_SOCKS_DATASOURCE_PROXY_CLIENT_CERT"
-	// PluginSecureSocksProxyClientKey is a constant for the GF_SECURE_SOCKS_DATASOURCE_PROXY_CLIENT_KEY
-	// environment variable used to specify the file location of the client key for the secure socks proxy.
-	PluginSecureSocksProxyClientKey = "GF_SECURE_SOCKS_DATASOURCE_PROXY_CLIENT_KEY"
-	// PluginSecureSocksProxyRootCACert is a constant for the GF_SECURE_SOCKS_DATASOURCE_PROXY_ROOT_CA_CERT
-	// environment variable used to specify the file location of the root ca for the secure socks proxy.
-	PluginSecureSocksProxyRootCACert = "GF_SECURE_SOCKS_DATASOURCE_PROXY_ROOT_CA_CERT"
-	// PluginSecureSocksProxyProxyAddress is a constant for the GF_SECURE_SOCKS_DATASOURCE_PROXY_PROXY_ADDRESS
-	// environment variable used to specify the secure socks proxy server address to proxy the connections to.
-	PluginSecureSocksProxyProxyAddress = "GF_SECURE_SOCKS_DATASOURCE_PROXY_PROXY_ADDRESS"
-	// PluginSecureSocksProxyServerName is a constant for the GF_SECURE_SOCKS_DATASOURCE_PROXY_SERVER_NAME
-	// environment variable used to specify the server name of the secure socks proxy.
-	PluginSecureSocksProxyServerName = "GF_SECURE_SOCKS_DATASOURCE_PROXY_SERVER_NAME"
-	// PluginSecureSocksProxyAllowInsecure is a constant for the GF_SECURE_SOCKS_DATASOURCE_PROXY_ALLOW_INSECURE
-	// environment variable used to specify if the proxy should use a TLS dialer.
+const (
+	PluginSecureSocksProxyEnabled       = "GF_SECURE_SOCKS_DATASOURCE_PROXY_SERVER_ENABLED"
```

Comment:
this one is not used it seems

Reply:
Here, no? https://github.com/grafana/grafana-plugin-sdk-go/blob/main/backend/config.go#L113

Reply:
oh, right, I only checked the PR code

Reply:
I'd come back to this then because we don't have a convention yet for any of these yet we have plenty of keys already at this stage 😅